lib/driver.js

```diff
@@ -771,12 +769,63 @@ class XCUITestDriver extends BaseDriver {
     }
     log.debug(`Setting initial orientation to '${orientation}'`);
     try {
-      await this.proxyCommand('/orientation', 'POST', {orientation});
+      await this._executeCommandWithTimeout(B.promisify(this.proxyCommand), 'setScreenOrientation', '/orientation', 'POST', {orientation});
       this.opts.curOrientation = orientation;
     } catch (err) {
       log.warn(`Setting initial orientation failed with: ${err}`);
     }
   }
+
+  _normalizeCommandTimeouts (value) {
```

Comment:
utils is maybe not the best place for it, since is not going to be reused anywhere else. I can just make it local module function (like 'static') instead of a class method